Python/ibmcloudsql/SQLQuery.py

```diff
@@ -311,57 +595,198 @@ def get_result(self, jobId, pagenumber=None):
 
         return result_df
 
-    def list_results(self, jobId):
-        self.logon()
+    def list_results(self, jobId, blocking=True):
```

Comment:
Same comment on blocking=True as I made below for rename_exact_result.

Reply:
Done in commit `1e12447646484b05db50f3f7923384379cede00`

Reply:
Hmm, seems you just renamed to wait here but did not change the method to call wait_for_job.